src/ORM/Marshaller.php

```diff
@@ -240,32 +240,22 @@ public function one(array $data, array $options = []): EntityInterface
      * Returns the validation errors for a data set based on the passed options
      *
      * @param array $data The data to validate.
-     * @param array<string, mixed> $options The options passed to this marshaller.
+     * @param bool|string $validator Validator name. `true` to use the default validator.
      * @param bool $isNew Whether it is a new entity or one to be updated.
      * @return array The list of validation errors.
      * @throws \RuntimeException If no validator can be created.
      */
-    protected function _validate(array $data, array $options, bool $isNew): array
+    protected function _validate(array $data, bool|string $validator, bool $isNew): array
```

Comment:
I would prefer the `Form::validate()` in #15815 also to be simple like this.

Reply:
Is false supposed to disable it? If not, should we use `null` to indicate default instead?

Reply:
> Is false supposed to disable it?

Yes, one can use `'validate' => false` in `$options` for `Table::newEntity()/patchEntity()` to disable validation.